_layouts/default.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>AgileFrance</title>
+
+		<link href="css/main.css" rel="stylesheet" type="text/css" media="all">
+	</head>
+	<body>
+		<header>
+			<h1><a href="/">Association Agile France</a></h1>
+			<img src="/logo.svg" height="150px" />
+		</header>
+
+		<main>
+			{{ content }}
+		</main>
+
+		<footer>
+			<p>
+				Association Agile France</br>
```

Comment:
Why not `<br>`?

Reply:
Because I did not change that part, and I expect this layout to be changed quite quickly through #2 😄